README.md

```diff
@@ -278,23 +278,7 @@ else:
 A couple of locations are checked, and we are happy to implement more if
 needed, just open an issue!
 
-Currently, it looks in the following places:
-- `__version__`
-- `version`
-- lookup `VERSION_ATTRIBUTES` in the scooby knowledge base
-- lookup `VERSION_METHODS` in the scooby knowledge base
-
-`VERSION_ATTRIBUTES` is a dictionary of attributes for known python packages
-with a non-standard place for the version, e.g. `VERSION_ATTRIBUTES['vtk'] =
-'VTK_VERSION'`. You can add other known places via:
-
-```py
-scooby.knowledge.VERSION_ATTRIBUTES['a_module'] = 'Awesome_version_location'
-```
-
-Similarly, `VERSION_METHODS` is a dictionary for methods to retrieve the
-version, and you can similarly add your methods which will get the version
-of a package.
+Currently, it uses `importlib.metadata.version` to get the distribution version.
```

Comment:
Maybe the entire section could be removed. Anyway, it is funny, as it will shift the focus of scooby. A big part of scooby was to find the version number in all potential odd situation. Which was sort of the detective part. Now, using importlib, scooby sort of (at least for the versions) is merely a display tool. Which is not a bad thing at all, but it is interesting to see where the whole thing came from and where it goes.